src/components/atoms/links/Link.tsx

```diff
@@ -22,13 +23,23 @@ const Link: React.FC<LinkProps> = ({
   dataCy,
   ariaLabelledBy,
   stopPropagation = false,
-  isHiddenFromScreenReaders
+  isHiddenFromScreenReaders,
+  trackClick
 }) => {
   const handleKeyUp = getLinkHandler({
     type: "keyup",
     isNewTab,
     stopPropagation,
-    url: href
+    url: href,
+    trackClick
+  });
+
+  const handleClick = getLinkHandler({
+    type: "click",
+    isNewTab,
+    stopPropagation,
+    url: href,
+    trackClick
```

Comment:
I see this reappearing from #1765. If it had been a shared commit there would be less risk of conflicts :smile: